website/docs/plugin/framework/migrating/attributes-blocks/fields.mdx

```diff
@@ -9,6 +9,38 @@ description: >-
+| Default               | [PlanModifiers](/framework/migrating/resources/plan-modification#framework) field on attribute or implementation of [ResourceWithModifyPlan](/plugin/framework/migrating/resources/plan-modification#framework) interface                                                                                                                                                                                                                                              |
+| DefaultFunc           | [PlanModifiers](/framework/migrating/resources/plan-modification#framework) field on attribute or implementation of [ResourceWithModifyPlan](/plugin/framework/migrating/resources/plan-modification#framework) interface                                                                                                                                                                                                                                              |
+| Description           | `Description` field on attribute                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| InputDefault          | [PlanModifiers](/framework/migrating/resources/plan-modification#framework) field on attribute or implementation of [ResourceWithModifyPlan](/plugin/framework/migrating/resources/plan-modification#framework) interface                                                                                                                                                                                                                                              |
+| StateFunc             | Requires implementation of bespoke logic before storing state, for instance in resource [Create method](plugin/framework/migrating/resources/crud#framework-1)                                                                                                                                                                                                                                                                                                         |
+| Elem                  | `ElementType` on [ListAttribute](/plugin/framework/migrating/attributes-blocks/types), [MapAttribute](/plugin/framework/migrating/attributes-blocks/types) or [SetAttribute](/plugin/framework/migrating/attributes-blocks/types)                                                                                                                                                                                                                                      |
```

Comment:
I think it might be good to call out that any `schema.Resource` in `Elem` should refer to the blocks documentation instead. 👍

Reply:
Updated to include call out.